controllers/profile_controller_test.go

```diff
@@ -36,10 +36,173 @@ var _ = Describe("ProfileController", func() {
 		Expect(k8sClient.Create(context.Background(), &nsp)).To(Succeed())
 	})
 
-	Context("Create", func() {
+	//Context("Create Local", func() {
```

Comment:
@Skarlso TODO: Uncomment this to see if things works once the multiple artifacts thing is handled.